### PR TITLE
footer page links has been fixed

### PR DESCRIPTION
**Pull request by uyo-obong**
Screenshot_2019-10-16_10-48-10
Screenshot_2019-10-16_10-47-56
Screenshot_2019-10-16_10-47-43
Screenshot_2019-10-16_10-47-23

### CODE REVIEW

**Comment by sulaimanxyz** [comment]
@uyo-obong Please review your codes and re-commit as discussed over the phone.
@Jamesjay4199 Please find time to review this PR.

**Comment by uyo-obong** [comment]
> @uyo-obong Please review your codes and re-commit as discussed over the phone.
> @Jamesjay4199 Please find time to review this PR.

done

File: packages/Webkul/Admin/src/Http/Controllers/ThemeManager/ThemeController.php [hunk 1]
```diff
@@ -45,4 +47,63 @@ public function update()
...
+
+                FooterPage::Create([
+                    "page_id" => $getPage[0],
+                    "name" => $getPage[1],
+                    "url" => $getPage[2]
+                ]);
+
+            }
+
+            DB::rollBack();
```
Comment (Jamesjay4199) [inline packages/Webkul/Admin/src/Http/Controllers/ThemeManager/ThemeController.php]: Why is the rollback here, I think it should be done in the catch block?

File: packages/Webkul/Admin/src/Http/routes.php [hunk 2]
```diff
@@ -502,6 +502,9 @@
                 ->name('admin.themes.store')
             ;
 
+            Route::post('/theme-manager/footer/pages/create', 'Webkul\Admin\Http\Controllers\ThemeManager\ThemeController@createNewFooterPages')->name('admin.configuration.footer.pages.create');
+
+
```
Comment (Jamesjay4199) [inline packages/Webkul/Admin/src/Http/routes.php]: No need for these multiple spaces

File: packages/Webkul/Admin/src/Http/Controllers/ConfigurationController.php [hunk 3]
```diff
@@ -2,10 +2,12 @@
 
 namespace Webkul\Admin\Http\Controllers;
 
+use App\FooterPage;
 use App\Page;
 use Illuminate\Http\Request;
 use Illuminate\Http\Response;
 use Illuminate\Support\Facades\Event;
+use PHPUnit\Framework\Constraint\Count;
```
Comment (Jamesjay4199) [inline packages/Webkul/Admin/src/Http/Controllers/ConfigurationController.php]: Is this needed here?

File: packages/Webkul/Admin/src/Http/Controllers/ConfigurationController.php [hunk 4]
```diff
@@ -2,10 +2,12 @@
 
 namespace Webkul\Admin\Http\Controllers;
 
+use App\FooterPage;
```
Comment (Jamesjay4199) [inline packages/Webkul/Admin/src/Http/Controllers/ConfigurationController.php]: Is this needed here?

Comment (uyo-obong) [reply packages/Webkul/Admin/src/Http/Controllers/ConfigurationController.php]: just resolved it.....

**Pull request merged**